introduction.tex

```diff
@@ -45,14 +45,31 @@ \section{Background}
 
 The three grand challenges can be summarized as a need to
 identify and plan for a ``most promising'' fuel cycle, while
-accurately calculating its impacts. 
-A simulation tool capable of accurately calculating the metrics
-of a fuel cycle transition scenario is essential for solving these challenges.
+accurately calculating its impacts.
+In 2011, the \gls{DOE} commissioned a report
+in order to plan the U.S. nuclear future.
+The report by Wigeland et al. identified potential fuel cycle groups and compared them
+to find the most `promising' fuel cycle \cite{wigeland_nuclear_2014}.
+The objective of the evaluation was to provide information about the
+potential benefits and challenges of nuclear fuel cycle options, in order to
+guide \gls{DOE} \gls{FCRD} program.
 
-This work will demonstrate capabilities of a system-level
-analysis tool, \Cyclus. The modeling capabilities of \Cyclus 
-will aid in the effort of solving
-the grand challenges mentioned above.
+However, this study was done for a static fuel cycle, where the
+material flow and reactor deployment are at equilibrium.
+Therefore the study did not take into account the dynamic
+changes in fuel demand and reactor deployment as well as the
+previously existing fleet. Thus, for a more comprehensive
+and realistic
+evaluation of a fuel cycle, fuel cycle analyses
```

Comment:
This is the crux of your argument and is a bit weak. What metrics are calculated poorly by static simulation and are calculated more accurately by transition scenarios? Are those metrics important to decision-making?

Reply:
has this been handled?

Reply:
I think I made it better.. If you have a suggestion to make it better let me know!